### PR TITLE
[WM-1612] Target workflow 4 pre-population

### PR DESCRIPTION
**Pull request by kpierre13**

### CODE REVIEW

File: service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+                  value: Github
+              - column:
+                  name: last_run_set_id
+                  value: null
+        - insert:
+            tableName: method_version
+            columns:
+              - column:
+                  name: method_version_id
+                  value: 40000000-0000-0000-0000-000000000004
```
Comment (mspector) [inline service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml:32]: is this a convention we're using for the version? just curious where this version ID comes from

**Review by mspector — APPROVED** [review]

File: service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                  name: record_type
+                  value: FOO
+
+        - update:
+            tableName: run_set
+            columns:
+              - column:
+                  name: method_version_id
+                  value: 40000000-0000-0000-0000-000000000004
+            where: run_set_id='10000000-0000-0000-0000-000000000004'
```
Comment (cjllanwarne) [inline service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml:227]: I might be wrong but this looks like maybe it shouldn't be necessary? The `insert` looks like it's already adding it with this method version ID?

**Review by cjllanwarne — APPROVED** [review]
One comment, otherwise LGTM

File: service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml (hunk 2 above)
Comment (kpierre13) [reply service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml:227]: I think you're right. This is redundant.

**Pull request merged**